torchrl/objectives/dreamer.py

```diff
@@ -342,6 +344,7 @@ def make_value_estimator(self, value_type: ValueEstimators = None, **hyperparams
             self._value_estimator = TDLambdaEstimator(
                 **hp,
                 value_network=value_net,
+                vectorized=False,  # TODO: vectorized version seems not to be similar to the non vectoried
```

Comment:
@BY571 I'm curious about this, it's supposed to be extensively tested in the CI

Reply:
I remember that I tested it like vec_value == value and there were some False mentioned. However, checking the values directly they were the same. This TODO can be taken off